apps/explorer/tsconfig.json

```diff
@@ -1,13 +1,19 @@
 {
+  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "baseUrl": "src",
     "jsx": "react-jsx",
     "allowJs": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "strict": true,
+    "types": ["node", "vite/client"],
+    "baseUrl": "src",
     "noImplicitAny": false,
-    "types": ["vite/client", "vitest", "node"]
+    "paths": {
+      "@cowprotocol/common-const": ["../../../libs/common-const/src/index.ts"],
+      "@cowprotocol/abis": ["../../../libs/abis/src/index.ts"],
+      "@cowprotocol/assets": ["../../../libs/assets/src/index.ts"]
+    }
```

Comment:
Do we need to map any lib in any other app that uses it?

Reply:
Unfortunately, I think we do :) 

Although is not updated (as it refers to CRACO), here are the notes I took when adding the first modules to the NX mono: https://www.notion.so/cownation/NX-CoW-Swap-Monorepo-38465f7efd7742069d57a13caab389a3?pvs=4#659a8fa163b348399a5d2705821a9c58

Here is more info on why this is necessary:
https://nx.dev/getting-started/tutorials/integrated-repo-tutorial#local-linking-of-packages

Reply:
Yes, Anxo is right